docs/manual/docs/administrator-guide/managing-users-and-groups/index.md

```diff
@@ -76,3 +76,126 @@ Rights associated with the roles are illustrated in detail in the list below:
+    | Preferred Records                           | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | Watch list                                  | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+    | "Sorted by relevancy,  modified,  title..." | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: | :material-check-circle: |
+
+=== "Metadata Page"
+ 
+    |                                             | Anonymous user          |     Registered User     | Editor| Reviewer|       User Admin        |     Admini- strator     |
+    |---------------------------------------------| :---: |:-----------------------:| :---: | :---: |:-----------------------:|:-----------------------:|
+    | Edit| | | :material-check-circle: | :material-check-circle: |          EDIT           | :material-check-circle: |
+    | Delete| | | :material-check-circle: | :material-check-circle: |           DEL           | :material-check-circle: |
+    | Cancel working copy| | | :material-check-circle: | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Privileges| | | :material-check-circle: | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Transfer Ownership| | | | |                         | :material-check-circle: |
+    | Manage Record / Unpublish| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Publish| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / submit for review| | | :material-check-circle: | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / directly approve metadata| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / approve metadata | | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / reject approval submision| | | | :material-check-circle: |                         | :material-check-circle: |
+    | Manage Record / Work flow / cancel approval submission| | | :material-check-circle: | |                         | :material-check-circle: |
+    | Manage Record / DOI Creation request| | | :material-check-circle: | :material-check-circle: |           UI            | :material-check-circle: |
+    | Manage Record / Duplicate| | | :material-check-circle: | :material-check-circle: |           UI            | :material-check-circle: |
```

Comment:
For `UserAdmin` think these options should not be in the UI, unless the user has also `Editor` / `Reviewer` profile.

Not really a problem of the documentation, but of the application.